troubleshooting.md

```diff
@@ -1385,3 +1385,22 @@ first process can acquire it, this type of `image not known` error can arise.
 The maintainers of Podman have considered heavier-duty locks to close this
 timing window. However, the slowdown that all Podman commands would encounter
 was not considered worth the cost of completely closing this small timing window.
+
+### 41) Podman fails to push image to insecure registry
+
+#### Symptom
+
+The registry runs with plan http instead of https, when podman push image such registry that has error: http: server gave HTTP response to HTTPS client.
+
+#### Solution
+
+1. cd /etc/containers/registries.conf.d
+2. create new file named `registry-namne.conf` and add content
+
+```
+[[registry]]
+location = "registry name"
+insecure = true
+```
+
+If you using mac, you should to execute `podman machine ssh`, login into podman machine before add insecure registy conf.
```

Comment:
As @edsantiago noted, this is number 4 on this list, but the solution differs.    Perhaps in the Solution section of number 4, you could append the following:

Or to avoid using the `--tls-verify` option:

1. cd /etc/containers/registries.conf.d
2. create a new file named `registry-name.conf` and add content

```
[[registry]]
location = "registry name"
insecure = true
```

If you are using a Mac, you should execute `podman machine ssh` and then login into `podman machine` before adding the insecure entry to the registry—conf file. As indicated in the entry, this is an insecure method and should be used cautiously.

Reply:
so we update the number 4 question ?

Reply:
Yes, please, and TY!

Reply:
update